master/buildbot/reporters/github.py

```diff
@@ -205,5 +216,6 @@ def createStatus(self,
         payload = {'body': description}
 
         return self._http.post(
-            '/'.join(['/repos', repo_user, repo_name, 'issues', issue, 'comments']),
+            '/'.join(['/repos', repo_user, repo_name,
```

Comment:
I formatted with autopep8. It seems to want to do this. I don't know why. I switch it back and did the format again .. this went back to the two line statement.

I can see about making the line length longer and seeing it this will go away